pkgs/tools/networking/openconnect/default.nix

```diff
@@ -14,21 +20,23 @@ stdenv.mkDerivation rec {
   };
 
   outputs = [ "out" "dev" ];
-
+  
   configureFlags = [
-    "--with-vpnc-script=${vpnc}/etc/vpnc/vpnc-script"
+    "--with-vpnc-script=${vpnc}/vpnc-script"
     "--disable-nls"
     "--without-openssl-version-check"
   ];
 
+  buildInputs = [ openssl gnutls gmp libxml2 stoken zlib ]
+    ++ stdenv.lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.PCSC;
   nativeBuildInputs = [ pkgconfig ];
-  propagatedBuildInputs = [ vpnc openssl gnutls gmp libxml2 stoken zlib ];
+  propagatedBuildInputs = [ vpnc ];
```

Comment:
This are just the vpnc sources, propagating them won't do anything.

Reply:
Ok. So would you put everything in nativeBuildInputs?

Reply:
I don't think vpnc needs to be added at all, since it doesn't contain any binaries or headers. Referencing it in configure is enough.

Reply:
@LnL7 Ok. I removed vpnc completely from buildInputs.